nuxeo-core-binarymanager-common/src/test/java/org/nuxeo/ecm/blob/AbstractTestCloudBinaryManager.java

```diff
@@ -83,7 +83,7 @@ protected void removeObject(String digest) throws IOException {
 
     @Before
     public void setUp() throws IOException {
-        assumeTrue("Cannot set Unlimited JCE Policy", setUnlimitedJCEPolicy());
+        //assumeTrue("Cannot set Unlimited JCE Policy", setUnlimitedJCEPolicy());
```

Comment:
You can remove the commented line. This hack won't work anymore, no need to keep this code.